pkg/api/latest/latest.go

```diff
@@ -67,6 +67,7 @@ var userResources = []string{"rc", "svc", "pods", "pvc"}
 func init() {
 	// Use the first API version in the list of registered versions as the latest.
 	Version = registered.RegisteredVersions[0]
+	OldestVersion = registered.RegisteredVersions[len(registered.RegisteredVersions)-1]
```

Comment:
There is now api/registered that registers api versions in the desired order: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/registered/registered.go#L41